OpencartTest/src/com/opencart/tools/RegexUtils.java

```diff
@@ -0,0 +1,77 @@
+package com.opencart.tools;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class RegexUtils {
+
+    private final static String DOUBLE_UI_SEPARATOR = ",";
+    private final static String PATTERN_UNSIGNED_NUMBER = "\\d+";
+    // private final static String PATTERN_UNSIGNED_DOUBLE = "\\d+(\\.\\d+)*";
+    private final static String PATTERN_UNSIGNED_DOUBLE = "\\d+(,\\d+)*\\.\\d{2}";
+    public final static String DELETE_SERVER = "http://[^/]+/";
+    private final static String EXTRACT_NUMBER_MESSAGE = "NumberFormatException for pattern =  %s text =  %s";
+
+    private RegexUtils() {
+    }
+
+    public static boolean isTextMatches(String pattern, String text) {
+        Matcher matcher = Pattern.compile(pattern).matcher(text);
```

Comment:
folks, read about Matchers and CoreMatchers
http://hamcrest.org/JavaHamcrest/tutorial